### PR TITLE
ci: Add initial CI scripts

### PR DESCRIPTION
**Pull request by jcvenegas**
Add base scripts to setup, run, and teardown tests
for ci.

Fixes: #21

Signed-off-by: Jose Carlos Venegas Munoz <jose.carlos.venegas.munoz@intel.com>

### CODE REVIEW

**Comment by jodh-intel** [comment]
lgtm

File: .ci/run.sh [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+# Copyright (c) 2017-2018 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+# This script will execute packaing tests suite
```
Comment (chavafg) [inline .ci/run.sh]: s/packaing/packaging

File: .ci/run.sh [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+# TODO: Add steps neded to build packages
```
Comment (chavafg) [inline .ci/run.sh]: s/neded/needed

**Comment by jcvenegas** [comment]
@chavafg branch updated.

**Pull request merged**